################################################################################
SHORT DESCRIPTION:
################################################################################
Enable building blktap3 from source installed within the Xen tools tree

################################################################################
LONG DESCRIPTION:
################################################################################
The blktap3 build from source cloned into the Xen tree produces library files
within a '.libs' subdirectory, so the Xen build rules need pointing to there.

################################################################################
CHANGELOG
################################################################################
Authors:
Christopher Clark <christopher.clark6@baesystems.com>

################################################################################
PATCHES
################################################################################
--- a/tools/Rules.mk
+++ b/tools/Rules.mk
@@ -177,8 +177,8 @@ endif
 ifeq ($(CONFIG_BLKTAP2),y)
 CFLAGS_libblktapctl = -I$(XEN_BLKTAP2)/control -I$(XEN_BLKTAP2)/include $(CFLAGS_xeninclude)
 SHDEPS_libblktapctl =
-LDLIBS_libblktapctl ?= $(SHDEPS_libblktapctl) $(XEN_BLKTAP2)/control/libblktapctl$(libextension)
-SHLIB_libblktapctl  = $(SHDEPS_libblktapctl) -Wl,-rpath-link=$(XEN_BLKTAP2)/control
+LDLIBS_libblktapctl ?= $(SHDEPS_libblktapctl) $(XEN_BLKTAP2)/control/.libs/libblktapctl$(libextension)
+SHLIB_libblktapctl  = $(SHDEPS_libblktapctl) -Wl,-rpath-link=$(XEN_BLKTAP2)/control/.libs
 else
 CFLAGS_libblktapctl =
 SHDEPS_libblktapctl =
